<?xml version="1.0" encoding="UTF-8"?>
    <ui version="4.0">
     <class>ECVTCategory</class>
     <widget class="QWidget" name="ECVTCategory">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>644</width>
        <height>685</height>
       </rect>
      </property>
      <property name="windowTitle">
       <string>Form</string>
      </property>
      <layout class="QGridLayout">
           <item row = "0" column = "0" >
    <widget class="QPushButton" name="btn_general">
     <property name="text">
      <string>General</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row = "1" column = "0" >
    <widget class="QPushButton" name="btn_maintenance">
     <property name="text">
      <string>Maintenance</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>88</width>
       <height>351</height>
      </size>
     </property>
    </spacer>
   </item> 
      <item row="4" column="0">
    <widget class="QPushButton" name="btn_close">
     <property name="text">
      <string>Close</string>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
      </layout>
     </widget>
     <resources/>
     <connections/>
    </ui>
    